phpunit.xml

```diff
@@ -20,7 +20,7 @@
     </filter>
     <php>
         <env name="APP_ENV" value="testing"/>
-        <env name="DB_DATABASE" value="starapi-testing"/>
+        <env name="DB_DATABASE" value="testing"/>
```

Comment:
revert this, db name for tests is `starapi-testing` - you even have that in test call routes below